### PR TITLE
Notif subs

### PR DESCRIPTION
**Pull request by Vindexus**

### CODE REVIEW

File: src/js/events.js [hunk 1]
```diff
@@ -137,41 +137,34 @@ function transformEventsPage (obj) {
...
-
-    //Has Opted In
-    //obj.emailOptIns = {eventsNearMe: true}
-
-    //Has Opted Out
-    //obj.emailOptIns = {eventsNearMe: false}
-
-    //Hasn't opted in or out
-    //obj.emailOptIns = {}
+    console.log('obj',obj);
```
Comment (tmathews) [inline src/js/events.js]: :hammer_and_pick: :evergreen_tree: Let's build a fort.

File: src/js/helpers.js [hunk 2]
```diff
@@ -315,6 +315,15 @@ function youTubeUserToChannelID (user, done) {
   })
 }
 
+function terror (err) {
```
Comment (tmathews) [inline src/js/helpers.js]: Lol what?

Comment (tmathews) [reply src/js/helpers.js]: Oh. I dunno man. :confounded:

Comment (Vindexus) [reply src/js/helpers.js]: Everywhere that we have 

```
if (err) {
  toasty(new Error(err))
  return
}```

becomes

```
if (terror(err)) {
  return
}
```

It's awesome. I can change the name if you want. `if (displayError(err))` or `if (showError(err)` or `if (errAndShow(err)` or something.

Comment (tmathews) [reply src/js/helpers.js]: Rename for sure.

Comment (Vindexus) [reply src/js/helpers.js]: Done.

**Pull request closed without merging**